<!DOCTYPE html>
<html lang="en">
    <head>
        <meta charset="utf-8">
        <meta http-equiv="X-UA-Compatible" content="IE=edge">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        
        <meta name="author" content="Guangchuang Yu">
        <link rel="canonical" href="https://guangchuangyu.github.io/GOSemSim/featuredArticles/">
        <link rel="shortcut icon" href="../img/favicon.ico">

	<title>Featured Articles - GOSemSim</title>

        <link href="../css/bootstrap-custom.min.css" rel="stylesheet">
        <link href="../css/font-awesome-4.0.3.css" rel="stylesheet">
        <link rel="stylesheet" href="../css/highlight.css">
        <link href="../css/base.css" rel="stylesheet">

        <!-- HTML5 shim and Respond.js IE8 support of HTML5 elements and media queries -->
        <!--[if lt IE 9]>
            <script src="https://oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js"></script>
            <script src="https://oss.maxcdn.com/libs/respond.js/1.3.0/respond.min.js"></script>
        <![endif]-->

        
    </head>

    <body>

        <div class="navbar navbar-default navbar-fixed-top" role="navigation">
    <div class="container">

        <!-- Collapsed navigation -->
        <div class="navbar-header">
            <!-- Expander button -->
            <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse">
                <span class="sr-only">Toggle navigation</span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
            </button>

            <!-- Main title -->
            <a class="navbar-brand" href="..">GOSemSim</a>
        </div>

        <!-- Expanded navigation -->
        <div class="navbar-collapse collapse">
            <!-- Main navigation -->
            <ul class="nav navbar-nav">
            
            
                <li >
                    <a href="..">Home</a>
                </li>
            
            
            
                <li >
                    <a href="../documentation/">Documentation</a>
                </li>
            
            
            
                <li class="active">
                    <a href="./">Featured Articles</a>
                </li>
            
            
            
                <li class="dropdown">
                    <a href="#" class="dropdown-toggle" data-toggle="dropdown">Software <b class="caret"></b></a>
                    <ul class="dropdown-menu">
                    
                        
<li >
    <a href="../ChIPseeker/">ChIPseeker</a>
</li>

                    
                        
<li >
    <a href="../clusterProfiler/">clusterProfiler</a>
</li>

                    
                        
<li >
    <a href="../DOSE/">DOSE</a>
</li>

                    
                        
<li >
    <a href="../emojifont/">emojifont</a>
</li>

                    
                        
<li >
    <a href="../ggtree/">ggtree</a>
</li>

                    
                        
<li >
    <a href="../GOSemSim/">GOSemSim</a>
</li>

                    
                        
<li >
    <a href="../meshes/">meshes</a>
</li>

                    
                        
<li >
    <a href="../ReactomePA/">ReactomePA</a>
</li>

                    
                        
<li >
    <a href="../treeio/">treeio</a>
</li>

                    
                    </ul>
                </li>
            
            
            </ul>

            <!-- Search, Navigation and Repo links -->
            <ul class="nav navbar-nav navbar-right">
                <li>
                    <a href="#" data-toggle="modal" data-target="#mkdocs_search_modal">
                        <i class="fa fa-search"></i> Search
                    </a>
                </li>
                <li >
                    <a rel="next" href="../documentation/">
                        <i class="fa fa-arrow-left"></i> Previous
                    </a>
                </li>
                <li >
                    <a rel="prev" href="../ChIPseeker/">
                        Next <i class="fa fa-arrow-right"></i>
                    </a>
                </li>
                
                <li>
                    <a href="https://github.com/GuangchuangYu/GOSemSim">
                        
                            <i class="fa fa-github"></i>
                        
                        GitHub
                    </a>
                </li>
                
            </ul>
        </div>
    </div>
</div>

        <div class="container">
            <div class="col-md-3"><div class="bs-sidebar hidden-print affix well" role="complementary">
    <ul class="nav bs-sidenav">
    
        <li class="main active"><a href="#disease-or-drug-analysis">Disease or Drug analysis</a></li>
        
    
        <li class="main "><a href="#geneprotein-functional-analysis">Gene/Protein functional analysis</a></li>
        
    
        <li class="main "><a href="#expression-data-processing">Expression data processing</a></li>
        
    
        <li class="main "><a href="#interactions">Interactions</a></li>
        
            <li><a href="#genetic-interaction">Genetic interaction</a></li>
        
            <li><a href="#protein-protein-interaction">Protein-Protein Interaction</a></li>
        
            <li><a href="#mirna-mrna-interaction">miRNA-mRNA Interaction</a></li>
        
    
        <li class="main "><a href="#cellular-localization">Cellular localization</a></li>
        
    
        <li class="main "><a href="#motif-analysis">Motif analysis</a></li>
        
    
        <li class="main "><a href="#non-coding-rna">non-coding RNA</a></li>
        
    
        <li class="main "><a href="#semantic-similarity-analysis">Semantic Similarity analysis</a></li>
        
    
        <li class="main "><a href="#evolution">Evolution</a></li>
        
    
    </ul>
</div></div>
            <div class="col-md-9" role="main">

<!-- AddToAny BEGIN -->

<div class="a2a_kit a2a_kit_size_32 a2a_default_style">

<a class="a2a_dd" href="//www.addtoany.com/share"></a>
<a class="a2a_button_facebook"></a> <a class="a2a_button_twitter"></a>
<a class="a2a_button_google_plus"></a>
<a class="a2a_button_pinterest"></a> <a class="a2a_button_reddit"></a>
<a class="a2a_button_sina_weibo"></a> <a class="a2a_button_wechat"></a>
<a class="a2a_button_douban"></a>

</div>

<script async src="//static.addtoany.com/menu/page.js"></script>

<!-- AddToAny END -->

<p><link rel="stylesheet" href="https://guangchuangyu.github.io/css/font-awesome.min.css">
<link rel="stylesheet" href="https://guangchuangyu.github.io/css/academicons.min.css"></p>
<p><a href="https://github.com/GuangchuangYu/featured_img">Let us know</a> if you have
published using <code>GOSemSim</code> and your publication will be featured here.</p>
<p><a href="https://scholar.google.com.hk/scholar?oi=bibs&amp;hl=en&amp;cites=9484177541993722322"><img alt="citation" src="https://img.shields.io/badge/cited%20by-273-blue.svg?style=flat" /></a>
<a href="http://apps.webofknowledge.com/InboundService.do?mode=FullRecord&amp;customersID=RID&amp;IsProductCode=Yes&amp;product=WOS&amp;Init=Yes&amp;Func=Frame&amp;DestFail=http%3A%2F%2Fwww.webofknowledge.com&amp;action=retrieve&amp;SrcApp=RID&amp;SrcAuth=RID&amp;SID=Y2CXu6nry8nDQZcUy1w&amp;UT=WOS%3A000276045800023"><img alt="" src="https://img.shields.io/badge/ESI-Highly%20Cited%20Paper-blue.svg?style=flat" /></a></p>
<p><link rel='stylesheet' href=https://guangchuangyu.github.io/resume/css/morris.css>
<script src='https://guangchuangyu.github.io/resume/css/jquery.min.js' type='text/javascript'></script>
<script src='https://guangchuangyu.github.io/resume/css/raphael-min.js' type='text/javascript'></script>
<script src='https://guangchuangyu.github.io/resume/css/morris-0.4.2.min.js' type='text/javascript'></script>
<style>
  .rChart {
    display: block;
    margin-left: auto; 
    margin-right: auto;
    width: 800px;
    height: 300px;
  }<br />
  </style>
<div id="chart155b21fa8c31" class="rChart morris"></p>
</div>

<script type='text/javascript'>
    var chartParams = {
 "element": "chart155b21fa8c31",
"width":            800,
"height":            400,
"xkey": "year",
"ykeys": [
 "cites" 
],
"data": [
 {
 "year": 2010,
"cites":              6,
"pubid": "tuHXwOkdijsC" 
},
{
 "year": 2011,
"cites":              9,
"pubid": "tuHXwOkdijsC" 
},
{
 "year": 2012,
"cites":             34,
"pubid": "tuHXwOkdijsC" 
},
{
 "year": 2013,
"cites":             42,
"pubid": "tuHXwOkdijsC" 
},
{
 "year": 2014,
"cites":             44,
"pubid": "tuHXwOkdijsC" 
},
{
 "year": 2015,
"cites":             48,
"pubid": "tuHXwOkdijsC" 
},
{
 "year": 2016,
"cites":             72,
"pubid": "tuHXwOkdijsC" 
},
{
 "year": 2017,
"cites":             18,
"pubid": "tuHXwOkdijsC" 
} 
],
"id": "chart155b21fa8c31",
"labels": "cites" 
},
      chartType = "Bar"
    new Morris[chartType](chartParams)
</script>

<p>Find out more on <a href="https://scholar.google.com.hk/scholar?oi=bibs&amp;hl=en&amp;cites=9484177541993722322"><i class="ai ai-google-scholar"></i> google
scholar</a>.</p>
<h2 id="disease-or-drug-analysis">Disease or Drug analysis</h2>
<ul>
<li><a href="http://dx.doi.org/10.4049/jimmunol.1501834">Regulatory T Cells Orchestrate Similar Immune Evasion of Fetuses
    and Tumors in Mice</a>.
    <strong>The Journal of Immunology</strong>. 2016, 196(2):678-690.</li>
<li><a href="http://bioinformatics.oxfordjournals.org/content/31/4/608.short">DOSE: an R/Bioconductor package for disease ontology semantic and
    enrichment
    analysis</a>.
    <strong>Bioinformatics</strong>. 2015, 31(4):608-609.</li>
<li><a href="http://nar.oxfordjournals.org/content/43/W1/W283.short">TFmiR: a web server for constructing and analyzing disease-specific
    transcription factor and miRNA co-regulatory
    networks</a>.
    <strong>Nucleic Acids Research</strong>. 2015, 43(W1):W283-W288.</li>
<li><a href="http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1003073">Human Monogenic Disease Genes Have Frequently Functionally
    Redundant
    Paralogs</a>.
    <strong>PLoS Computational Biology</strong>. 2013, 9(5):e1003073.</li>
<li><a href="http://10.1093/nar/gkw1270">Flexible model-based clustering of mixed binary and continuous
    data: application to genetic regulation and
    cancer</a>. <strong>Nucleic Acids Research</strong>.
    2017</li>
</ul>
<h2 id="geneprotein-functional-analysis">Gene/Protein functional analysis</h2>
<ul>
<li><a href="http://dx.doi.org/10.1186/s13024-016-0095-2">Network-driven plasma proteomics expose molecular changes in the
    Alzheimer’s brain</a>.
    <strong>Molecular Neurodegeneration</strong>. 2016, 11:31.</li>
<li><a href="http://dx.doi.org/10.1371/journal.pcbi.1004892">Single-Cell Co-expression Analysis Reveals Distinct Functional
    Modules, Co-regulation Mechanisms and Clinical
    Outcomes</a>. <strong>PLOS
    Computational Biology</strong>. 2016, 12(4):e1004892.</li>
<li><a href="http://dx.doi.org/10.1039/C5MB00881F%3E">Crosstalk of dynamic functional modules in lung development of
    rhesus macaques</a>.
    <strong>Mol. Biosyst.</strong>. 2016, 12:1342-1349.</li>
<li><a href="http://dx.doi.org/10.1186/s13059-016-0902-7">Comparative transcriptomics reveals the conserved building blocks
    involved in parallel evolution of diverse phenotypic traits in
    ants</a>. <strong>Genome
    Biology</strong>. 2016, 17:43.</li>
<li><a href="http://bioinformatics.oxfordjournals.org/content/31/11/1857">protr/ProtrWeb: R package and web server for generating various
    numerical representation schemes of protein
    sequences</a>.
    <strong>Bioinformatics</strong>. 2015, 31(11):1857-1859.</li>
<li><a href="http://bioinformatics.oxfordjournals.org/content/30/2/289.short">tRanslatome: an R/Bioconductor package to portray translational
    control</a>.
    <strong>Bioinformatics</strong>. 2014, 30(2):289-291.</li>
<li><a href="https://nar.oxfordjournals.org/content/early/2014/05/21/nar.gku442.short">EvoCor: a platform for predicting functionally related genes using
    phylogenetic and expression
    profiles</a>.
    <strong>Nucleic Acids Research</strong>. 2014, 42(W1):W72-W75.</li>
<li><a href="http://www.pnas.org/content/111/13/4892.short">Genome-wide activity of unliganded estrogen receptor-α in breast
    cancer cells</a>. <strong>Proc
    Natl Acad Sci</strong>. 2014, 111(13):4892-4897.</li>
<li><a href="http://www.plantcell.org/content/25/7/2714.short">Constitutively Elevated Salicylic Acid Levels Alter Photosynthesis
    and Oxidative State but Not Growth in Transgenic
    <em>Populus</em></a>. <strong>The
    Plant Cell</strong>. 2013, 25(7):2714-2730.</li>
</ul>
<h2 id="expression-data-processing">Expression data processing</h2>
<ul>
<li><a href="http://dx.doi.org/10.4172/jpb.1000387">Robust Detection of Outlier Samples and Genes in Expression
    Datasets</a>. <strong>Journal of
    Proteomics &amp; Bioinformatics</strong>. 2016, 9:38-48.</li>
<li><a href="http://dx.doi.org/10.1186/s12859-015-0853-0%3E">Missing value imputation for microRNA expression data by using a
    GO-based similarity
    measure</a>. <strong>BMC
    Bioinformatics</strong>. 2016, 17:10.</li>
</ul>
<h2 id="interactions">Interactions</h2>
<h3 id="genetic-interaction">Genetic interaction</h3>
<ul>
<li><a href="https://www.ncbi.nlm.nih.gov/pubmed/28406896">Inferring modulators of genetic interactions with epistatic nested
    effects models</a>.
    <strong>PLoS Comput Biol.</strong>. 2017, 13(4):e1005496</li>
</ul>
<h3 id="protein-protein-interaction">Protein-Protein Interaction</h3>
<ul>
<li><a href="http://www.nature.com/nprot/journal/v11/n10/abs/nprot.2016.117.html">Analyzing and interpreting genome data at the network level with
    ConsensusPathDB</a>.
    <strong>Nature Protocols</strong>. 2016, 11:1889-1907.</li>
<li><a href="http://dx.doi.org/10.1016/j.jtbi.2016.05.020%3E">Integration of multiple biological features yields high confidence
    human protein
    interactome</a>.
    <strong>Journal of Theoretical Biology</strong>. 2016, 403:85-96.</li>
<li><a href="http://dx.doi.org/10.1039/C6MB00065G%3E">Computational prediction of virus–human protein–protein
    interactions using embedding kernelized heterogeneous
    data</a>. <strong>Mol. BioSyst</strong>.
    2016, 12:1976-1986.</li>
<li><a href="http://dx.doi.org/10.1093/bioinformatics/btv528%3E">Computational probing protein–protein interactions targeting small
    molecules</a>.
    <strong>Bioinformatics</strong>. 2016, 32(2):226-234.</li>
<li><a href="http://dx.doi.org/10.1039/C5MB00417A%3E">An integrative <em>C. elegans</em> protein–protein interaction network
    with reliability assessment based on a probabilistic graphical
    model</a>. <strong>Mol. Biosyst</strong>.
    2016, 12:85-92.</li>
<li><a href="http://www.nature.com/articles/srep07702?WT.ec_id=SREP-20150113">A Highly Efficient Approach to Protein Interactome Mapping Based on
    Collaborative Filtering
    Framework</a>.
    <strong>Scientific Reports</strong>. 5:7702.</li>
<li><a href="http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1004202">Deciphering Signaling Pathway Networks to Understand the Molecular
    Mechanisms of Metformin
    Action</a>.
    <strong>PLoS Comput Biol</strong>. 2015, 11(6):e1004202.</li>
<li><a href="http://bioinformatics.oxfordjournals.org/content/29/3/355.short">A novel link prediction algorithm for reconstructing
    protein–protein interaction networks by topological
    similarity</a>.
    <strong>Bioinformatics</strong>. 2013, 29(3):355-364.</li>
<li><a href="http://bioinformatics.oxfordjournals.org/content/29/13/i199.short">Minimum curvilinearity to enhance topological prediction of protein
    interactions by network
    embedding</a>.
    <strong>Bioinformatics</strong>. 2013, 29(13):i199-i209.</li>
<li><a href="https://nar.oxfordjournals.org/content/40/W1/W140.short">IntScore: a web tool for confidence scoring of biological
    interactions</a>.
    <strong>Nucl. Acids Res.</strong>. 2012, 40(W1):W140-W146.</li>
</ul>
<h3 id="mirna-mrna-interaction">miRNA-mRNA Interaction</h3>
<ul>
<li>
<p><a href="http://www.pnas.org/content/early/2016/12/02/1612024114.full">miR-17∼92 family clusters control iNKT cell ontogenesis via
    modulation of TGF-β
    signaling</a>.
    <strong>PNAS</strong>. 2016.</p>
</li>
<li>
<p><a href="http://dx.doi.org/10.1016/j.jtbi.2016.05.026%3E">Identifying Functional cancer-specific miRNA-mRNA interactions in
    testicular germ cell
    tumor</a>. <strong>Journal
    of Theoretical Biology</strong>. 2016, 404:82-96.</p>
</li>
<li><a href="http://bioinformatics.oxfordjournals.org/content/31/14/2403">miR2GO: comparative functional analysis for
    microRNAs</a>.
    <strong>Bioinformatics</strong>. 2015, 31(14):2403-2405.</li>
<li><a href="http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1002488">Uncovering MicroRNA and Transcription Factor Mediated Regulatory
    Networks in
    Glioblastoma</a>.
    <strong>PLoS Comput Biol</strong>. 2012, 8(7):e1002488.</li>
<li><a href="https://bib.oxfordjournals.org/content/12/6/588.short">myMIR: a genome-wide microRNA targets identification and annotation
    tool</a>.
    <strong>Briefings in Bioinformatics</strong>. 2011, 12(6):588-600.</li>
<li><a href="https://jclinbioinformatics.biomedcentral.com/articles/10.1186/2043-9113-1-15">Functional similarity analysis of human virus-encoded
    miRNAs</a>.
    <strong>Journal of Clinical Bioinformatics</strong>. 2011, 1:15.</li>
</ul>
<h2 id="cellular-localization">Cellular localization</h2>
<ul>
<li><a href="http://dx.doi.org/10.1093/bioinformatics/btw723">Hum-mPLoc 3.0: Prediction enhancement of human protein subcellular
    localization through modeling the hidden correlations of gene
    ontology and functional domain
    features</a>.
    <strong>Bioinformatics</strong>. 2017.</li>
</ul>
<h2 id="motif-analysis">Motif analysis</h2>
<ul>
<li><a href="http://dx.doi.org/10.1093/bib/bbw063">Comparative pan-cancer DNA methylation analysis reveals cancer
    common and specific patterns</a>.
    <strong>Brief Bioinform</strong>, 2016</li>
</ul>
<h2 id="non-coding-rna">non-coding RNA</h2>
<ul>
<li><a href="http://nar.oxfordjournals.org/content/early/2016/10/12/nar.gkw909.full">Global and cell-type specific properties of lincRNAs with ribosome
    occupancy</a>.
    ** Nucl. Acids Res**. 2016.</li>
<li><a href="https://bib.oxfordjournals.org/content/early/2014/12/03/bib.bbu045.short">Advantages of mixing bioinformatics and visualization approaches
    for analyzing sRNA-mediated regulatory bacterial
    networks</a>.
    <strong>Briefings In Bioinformatics</strong>. 2015, 16(5):795-805.</li>
</ul>
<h2 id="semantic-similarity-analysis">Semantic Similarity analysis</h2>
<ul>
<li><a href="http://bioinformatics.oxfordjournals.org/content/32/3/477.short">A-DaGO-Fun: an adaptable Gene Ontology semantic similarity-based
    functional analysis
    tool</a>.
    <strong>Bioinformatics</strong>. 2016, 32(3):477-479.</li>
<li><a href="http://bioinformatics.oxfordjournals.org/content/30/5/740.short">The semantic measures library and toolkit: fast computation of
    semantic similarity and relatedness using biomedical
    ontologies</a>.
    <strong>Bioinformatics</strong>. 2014, 30(5):740-742.</li>
<li><a href="https://bib.oxfordjournals.org/content/13/5/569.short">Semantic similarity analysis of protein data: assessment with
    biological features and
    issues</a>.
    <strong>Briefings in Bioinformatics</strong>. 2012, 13(5):569-585.</li>
</ul>
<h2 id="evolution">Evolution</h2>
<ul>
<li><a href="http://dx.doi.org/10.1101/gr.202200.115">Venus flytrap carnivorous lifestyle builds on herbivore defense
    strategies</a>. <strong>Cold Spring
    Harbor Laboratory Press</strong>. 2016, 26:812-825.</li>
<li><a href="http://dx.doi.org/10.1093/nar/gkv1274">FunTree: advances in a resource for exploring and contextualising
    protein function evolution</a>,
    <strong>Nucl. Acids Res.</strong>. 2016, 44(D1):D317-D323.</li>
<li><a href="http://genome.cshlp.org/content/22/4/714.short">Evolutionary rate covariation reveals shared functionality and
    coexpression of
    genes</a>. <strong>Genome
    Research</strong>. 2012, 22:714-720.</li>
</ul></div>
        </div>

        <footer class="col-md-12">
            <hr>
            
                <center>Copyright &copy; 2016 <a href="https://guangchuangyu.github.io">Guangchuang Yu</a>.<div style="width:250px;margin:auto;"><script type="text/javascript" src="//rf.revolvermaps.com/0/0/7.js?i=5qpvv5zz7fp&amp;m=0&amp;c=ff0000&amp;cr1=ffffff&amp;sx=0" async="async"></script></div></center>
            
            <center>Documentation built with <a href="http://www.mkdocs.org/">MkDocs</a>.</center>
        </footer>

        <script src="../js/jquery-1.10.2.min.js"></script>
        <script src="../js/bootstrap-3.0.3.min.js"></script>
        <script src="../js/highlight.pack.js"></script>
        <script>var base_url = '..';</script>
        <script data-main="../mkdocs/js/search.js" src="../mkdocs/js/require.js"></script>
        <script src="../js/base.js"></script>

        <div class="modal" id="mkdocs_search_modal" tabindex="-1" role="dialog" aria-labelledby="Search Modal" aria-hidden="true">
            <div class="modal-dialog">
                <div class="modal-content">
                    <div class="modal-header">
                        <button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span><span class="sr-only">Close</span></button>
                        <h4 class="modal-title" id="exampleModalLabel">Search</h4>
                    </div>
                    <div class="modal-body">
                        <p>
                            From here you can search these documents. Enter
                            your search terms below.
                        </p>
                        <form role="form">
                            <div class="form-group">
                                <input type="text" class="form-control" placeholder="Search..." id="mkdocs-search-query">
                            </div>
                        </form>
                        <div id="mkdocs-search-results"></div>
                    </div>
                    <div class="modal-footer">
                    </div>
                </div>
            </div>
        </div>
    </body>
</html>