---
date: 2017-07-24T00:11:02+01:00
title: Featured Articles
weight: 30
---



<p><link rel="stylesheet" href="https://guangchuangyu.github.io/css/font-awesome.min.css"> <link rel="stylesheet" href="https://guangchuangyu.github.io/css/academicons.min.css"></p>
<p><a href="https://github.com/GuangchuangYu/featured_img">Let us know</a> if you have published using <code>GOSemSim</code> and your publication will be featured here.</p>
<p><img src="../images/citation.png" width="880"/></p>
<div id="disease-or-drug-analysis" class="section level2">
<h2>Disease or Drug analysis</h2>
<ul>
<li><a href="http://dx.doi.org/10.4049/jimmunol.1501834">Regulatory T Cells Orchestrate Similar Immune Evasion of Fetuses and Tumors in Mice</a>. <strong>The Journal of Immunology</strong>. 2016, 196(2):678-690.</li>
<li><a href="http://bioinformatics.oxfordjournals.org/content/31/4/608.short">DOSE: an R/Bioconductor package for disease ontology semantic and enrichment analysis</a>. <strong>Bioinformatics</strong>. 2015, 31(4):608-609.</li>
<li><a href="http://nar.oxfordjournals.org/content/43/W1/W283.short">TFmiR: a web server for constructing and analyzing disease-specific transcription factor and miRNA co-regulatory networks</a>. <strong>Nucleic Acids Research</strong>. 2015, 43(W1):W283-W288.</li>
<li><a href="http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1003073">Human Monogenic Disease Genes Have Frequently Functionally Redundant Paralogs</a>. <strong>PLoS Computational Biology</strong>. 2013, 9(5):e1003073.</li>
<li><a href="http://10.1093/nar/gkw1270">Flexible model-based clustering of mixed binary and continuous data: application to genetic regulation and cancer</a>. <strong>Nucleic Acids Research</strong>. 2017</li>
</ul>
</div>
<div id="geneprotein-functional-analysis" class="section level2">
<h2>Gene/Protein functional analysis</h2>
<ul>
<li><a href="http://dx.doi.org/10.1186/s13024-016-0095-2">Network-driven plasma proteomics expose molecular changes in the Alzheimer’s brain</a>. <strong>Molecular Neurodegeneration</strong>. 2016, 11:31.</li>
<li><a href="http://dx.doi.org/10.1371/journal.pcbi.1004892">Single-Cell Co-expression Analysis Reveals Distinct Functional Modules, Co-regulation Mechanisms and Clinical Outcomes</a>. <strong>PLOS Computational Biology</strong>. 2016, 12(4):e1004892.</li>
<li><a href="http://dx.doi.org/10.1039/C5MB00881F%3E">Crosstalk of dynamic functional modules in lung development of rhesus macaques</a>. <strong>Mol. Biosyst.</strong>. 2016, 12:1342-1349.</li>
<li><a href="http://dx.doi.org/10.1186/s13059-016-0902-7">Comparative transcriptomics reveals the conserved building blocks involved in parallel evolution of diverse phenotypic traits in ants</a>. <strong>Genome Biology</strong>. 2016, 17:43.</li>
<li><a href="http://bioinformatics.oxfordjournals.org/content/31/11/1857">protr/ProtrWeb: R package and web server for generating various numerical representation schemes of protein sequences</a>. <strong>Bioinformatics</strong>. 2015, 31(11):1857-1859.</li>
<li><a href="http://bioinformatics.oxfordjournals.org/content/30/2/289.short">tRanslatome: an R/Bioconductor package to portray translational control</a>. <strong>Bioinformatics</strong>. 2014, 30(2):289-291.</li>
<li><a href="https://nar.oxfordjournals.org/content/early/2014/05/21/nar.gku442.short">EvoCor: a platform for predicting functionally related genes using phylogenetic and expression profiles</a>. <strong>Nucleic Acids Research</strong>. 2014, 42(W1):W72-W75.</li>
<li><a href="http://www.pnas.org/content/111/13/4892.short">Genome-wide activity of unliganded estrogen receptor-α in breast cancer cells</a>. <strong>Proc Natl Acad Sci</strong>. 2014, 111(13):4892-4897.</li>
<li><a href="http://www.plantcell.org/content/25/7/2714.short">Constitutively Elevated Salicylic Acid Levels Alter Photosynthesis and Oxidative State but Not Growth in Transgenic <em>Populus</em></a>. <strong>The Plant Cell</strong>. 2013, 25(7):2714-2730.</li>
</ul>
</div>
<div id="expression-data-processing" class="section level2">
<h2>Expression data processing</h2>
<ul>
<li><a href="http://dx.doi.org/10.4172/jpb.1000387">Robust Detection of Outlier Samples and Genes in Expression Datasets</a>. <strong>Journal of Proteomics &amp; Bioinformatics</strong>. 2016, 9:38-48.</li>
<li><a href="http://dx.doi.org/10.1186/s12859-015-0853-0%3E">Missing value imputation for microRNA expression data by using a GO-based similarity measure</a>. <strong>BMC Bioinformatics</strong>. 2016, 17:10.</li>
</ul>
</div>
<div id="interactions" class="section level2">
<h2>Interactions</h2>
<div id="genetic-interaction" class="section level3">
<h3>Genetic interaction</h3>
<ul>
<li><a href="https://www.ncbi.nlm.nih.gov/pubmed/28406896">Inferring modulators of genetic interactions with epistatic nested effects models</a>. <strong>PLoS Comput Biol.</strong>. 2017, 13(4):e1005496</li>
</ul>
</div>
<div id="protein-protein-interaction" class="section level3">
<h3>Protein-Protein Interaction</h3>
<ul>
<li><a href="https://doi.org/10.1093/bib/bbx123">Critical assessment and performance improvement of plant–pathogen protein–protein interaction prediction methods</a>. *** Briefings in Bioinformatics***. 2017.</li>
<li><a href="http://www.nature.com/nprot/journal/v11/n10/abs/nprot.2016.117.html">Analyzing and interpreting genome data at the network level with ConsensusPathDB</a>. <strong>Nature Protocols</strong>. 2016, 11:1889-1907.</li>
<li><a href="http://dx.doi.org/10.1016/j.jtbi.2016.05.020%3E">Integration of multiple biological features yields high confidence human protein interactome</a>. <strong>Journal of Theoretical Biology</strong>. 2016, 403:85-96.</li>
<li><a href="http://dx.doi.org/10.1039/C6MB00065G%3E">Computational prediction of virus–human protein–protein interactions using embedding kernelized heterogeneous data</a>. <strong>Mol. BioSyst</strong>. 2016, 12:1976-1986.</li>
<li><a href="http://dx.doi.org/10.1093/bioinformatics/btv528%3E">Computational probing protein–protein interactions targeting small molecules</a>. <strong>Bioinformatics</strong>. 2016, 32(2):226-234.</li>
<li><a href="http://dx.doi.org/10.1039/C5MB00417A%3E">An integrative <em>C. elegans</em> protein–protein interaction network with reliability assessment based on a probabilistic graphical model</a>. <strong>Mol. Biosyst</strong>. 2016, 12:85-92.</li>
<li><a href="http://www.nature.com/articles/srep07702?WT.ec_id=SREP-20150113">A Highly Efficient Approach to Protein Interactome Mapping Based on Collaborative Filtering Framework</a>. <strong>Scientific Reports</strong>. 5:7702.</li>
<li><a href="http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1004202">Deciphering Signaling Pathway Networks to Understand the Molecular Mechanisms of Metformin Action</a>. <strong>PLoS Comput Biol</strong>. 2015, 11(6):e1004202.</li>
<li><a href="http://bioinformatics.oxfordjournals.org/content/29/3/355.short">A novel link prediction algorithm for reconstructing protein–protein interaction networks by topological similarity</a>. <strong>Bioinformatics</strong>. 2013, 29(3):355-364.</li>
<li><a href="http://bioinformatics.oxfordjournals.org/content/29/13/i199.short">Minimum curvilinearity to enhance topological prediction of protein interactions by network embedding</a>. <strong>Bioinformatics</strong>. 2013, 29(13):i199-i209.</li>
<li><a href="https://nar.oxfordjournals.org/content/40/W1/W140.short">IntScore: a web tool for confidence scoring of biological interactions</a>. <strong>Nucl. Acids Res.</strong>. 2012, 40(W1):W140-W146.</li>
</ul>
</div>
<div id="mirna-mrna-interaction" class="section level3">
<h3>miRNA-mRNA Interaction</h3>
<ul>
<li><p><a href="http://www.pnas.org/content/early/2016/12/02/1612024114.full">miR-17∼92 family clusters control iNKT cell ontogenesis via modulation of TGF-β signaling</a>. <strong>PNAS</strong>. 2016.</p></li>
<li><a href="http://dx.doi.org/10.1016/j.jtbi.2016.05.026%3E">Identifying Functional cancer-specific miRNA-mRNA interactions in testicular germ cell tumor</a>. <strong>Journal of Theoretical Biology</strong>. 2016, 404:82-96.</li>
<li><a href="http://bioinformatics.oxfordjournals.org/content/31/14/2403">miR2GO: comparative functional analysis for microRNAs</a>. <strong>Bioinformatics</strong>. 2015, 31(14):2403-2405.</li>
<li><a href="http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1002488">Uncovering MicroRNA and Transcription Factor Mediated Regulatory Networks in Glioblastoma</a>. <strong>PLoS Comput Biol</strong>. 2012, 8(7):e1002488.</li>
<li><a href="https://bib.oxfordjournals.org/content/12/6/588.short">myMIR: a genome-wide microRNA targets identification and annotation tool</a>. <strong>Briefings in Bioinformatics</strong>. 2011, 12(6):588-600.</li>
<li><p><a href="https://jclinbioinformatics.biomedcentral.com/articles/10.1186/2043-9113-1-15">Functional similarity analysis of human virus-encoded miRNAs</a>. <strong>Journal of Clinical Bioinformatics</strong>. 2011, 1:15.</p></li>
</ul>
</div>
</div>
<div id="cellular-localization" class="section level2">
<h2>Cellular localization</h2>
<ul>
<li><a href="http://dx.doi.org/10.1093/bioinformatics/btw723">Hum-mPLoc 3.0: Prediction enhancement of human protein subcellular localization through modeling the hidden correlations of gene ontology and functional domain features</a>. <strong>Bioinformatics</strong>. 2017.</li>
</ul>
</div>
<div id="motif-analysis" class="section level2">
<h2>Motif analysis</h2>
<ul>
<li><a href="http://dx.doi.org/10.1093/bib/bbw063">Comparative pan-cancer DNA methylation analysis reveals cancer common and specific patterns</a>. <strong>Brief Bioinform</strong>, 2016</li>
</ul>
</div>
<div id="non-coding-rna" class="section level2">
<h2>non-coding RNA</h2>
<ul>
<li><a href="http://nar.oxfordjournals.org/content/early/2016/10/12/nar.gkw909.full">Global and cell-type specific properties of lincRNAs with ribosome occupancy</a>. ** Nucl. Acids Res**. 2016.</li>
<li><a href="https://bib.oxfordjournals.org/content/early/2014/12/03/bib.bbu045.short">Advantages of mixing bioinformatics and visualization approaches for analyzing sRNA-mediated regulatory bacterial networks</a>. <strong>Briefings In Bioinformatics</strong>. 2015, 16(5):795-805.</li>
</ul>
</div>
<div id="semantic-similarity-analysis" class="section level2">
<h2>Semantic Similarity analysis</h2>
<ul>
<li><a href="http://bioinformatics.oxfordjournals.org/content/32/3/477.short">A-DaGO-Fun: an adaptable Gene Ontology semantic similarity-based functional analysis tool</a>. <strong>Bioinformatics</strong>. 2016, 32(3):477-479.</li>
<li><a href="http://bioinformatics.oxfordjournals.org/content/30/5/740.short">The semantic measures library and toolkit: fast computation of semantic similarity and relatedness using biomedical ontologies</a>. <strong>Bioinformatics</strong>. 2014, 30(5):740-742.</li>
<li><a href="https://bib.oxfordjournals.org/content/13/5/569.short">Semantic similarity analysis of protein data: assessment with biological features and issues</a>. <strong>Briefings in Bioinformatics</strong>. 2012, 13(5):569-585.</li>
</ul>
<div id="reducing-go-term-redundancy" class="section level3">
<h3>Reducing GO term redundancy</h3>
<ul>
<li><a href="http://dx.doi.org/10.1073/pnas.1715865115">Faulty neuronal determination and cell polarization are reverted by modulating HD early phenotypes</a> <strong><em>PNAS</em></strong>. 2017.</li>
<li><a href="https://doi.org/10.1038/s41467-017-02185-w">EGFR feedback-inhibition by Ran-binding protein 6 is disrupted in cancer</a> <strong><em>Nature Communications</em></strong>. 2017, 8(1):2035.</li>
<li><a href="http://advances.sciencemag.org/content/3/12/e1701211.full">Intracerebroventricular delivery of hematopoietic progenitors results in rapid and robust engraftment of microglia-like cells</a>. <strong><em>Science Advances</em></strong>. 2017, 3(12):e1701211.</li>
</ul>
</div>
</div>
<div id="evolution" class="section level2">
<h2>Evolution</h2>
<ul>
<li><a href="http://dx.doi.org/10.1101/gr.202200.115">Venus flytrap carnivorous lifestyle builds on herbivore defense strategies</a>. <strong>Cold Spring Harbor Laboratory Press</strong>. 2016, 26:812-825.</li>
<li><a href="http://dx.doi.org/10.1093/nar/gkv1274">FunTree: advances in a resource for exploring and contextualising protein function evolution</a>, <strong>Nucl. Acids Res.</strong>. 2016, 44(D1):D317-D323.</li>
<li><a href="http://genome.cshlp.org/content/22/4/714.short">Evolutionary rate covariation reveals shared functionality and coexpression of genes</a>. <strong>Genome Research</strong>. 2012, 22:714-720.</li>
</ul>
</div>
